src/views/check-your-answers.njk

```diff
@@ -19,14 +22,13 @@
 	          text: props.data.limitedPartnership.data.partnership_name
 	        },
 	        actions: {
-	          items: [
-	            {
-	              href: "javascript:void(0)",
-	              text: "Change",
-	              visuallyHiddenText: "Name"
-	            }
-	          ]
-	        }
+            items: [ CREATE_CHANGE_LINK(
+              i18n.checkYourAnswersPage.change,
+              currentUrl | replace("check-your-answers", "name#partnership_name"),
```

Comment:
to avoid repeating the replace each time you can create a function (Macro) on this template.
something like :
{% macro replaceUrlSegment(url, oldSegment, newSegment) %}
  {{ url | replace(oldSegment, newSegment) }}
{% endmacro %}

then you can call:
replaceUrlSegment(currentUrl, "check-your-answers", "name#partnership_name")
in the template

Reply:
Using a constant rather than a hardcoded value.